awx/main/models/inventory.py

```diff
@@ -1302,11 +1260,7 @@ def preferred_instance_groups(self):
         return selected_groups
 
     def cancel(self, job_explanation=None, is_chain=False):
-        res = super(InventoryUpdate, self).cancel(job_explanation=job_explanation, is_chain=is_chain)
-        if res:
-            if self.launch_type != 'scm' and self.source_project_update:
-                self.source_project_update.cancel(job_explanation=job_explanation)
-        return res
+        return super(InventoryUpdate, self).cancel(job_explanation=job_explanation, is_chain=is_chain)
```

Comment:
no need to keep this method, you can delete the entire `def` as the super class implementation is fine now.

Reply:
oh right, of course, thanks!